ShellScript/Bash.sublime-syntax

```diff
@@ -157,50 +157,50 @@ contexts:
   control:
     - match: \b(if){{keyword_boundary_end}}(?:\s*(!))?
       captures:
-        1: keyword.control.if.begin.shell
+        1: keyword.control.conditional.if.shell
         2: keyword.operator.logical.shell
     - match: \bthen\b
-      scope: keyword.control.if.then.shell
+      scope: keyword.control.conditional.then.shell
       pop: true
     - match: \b(elif){{keyword_boundary_end}}(?:\s*(!))?
       captures:
-        1: keyword.control.if.elif.shell
+        1: keyword.control.conditional.elif.shell
```

Comment:
If we're standardizing, this should probably be `elseif`. (https://github.com/sublimehq/Packages/issues/1228#issuecomment-337072664)